ote/src/cljs/ote/views/transport_service_common.cljs

```diff
@@ -151,16 +151,23 @@
                        :width "5%"
                        }
                       {:name ::t-service/format
-                       :type :string
+                       :type :chip-input
+                       :open-on-focus? true
+                       :suggestions ["GTFS" "Kalkati.net" "SIRI" "NeTEx" "GeoJSON" "JSON" "CSV"]
```

Comment:
Could be in separate array and not hard coded here. Would help if these are moved to database and administrated by admin in some day.

Reply:
Yes, currently both licenses and format suggestions are hardcoded. They can be added into database if we decide they should be admistrated.